tests/Command/ImportMeshUniverseCommandAcceptanceTest.php

```diff
@@ -0,0 +1,179 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\Command;
+
+use App\Entity\MeshConcept;
+use App\Entity\MeshDescriptor;
+use App\Entity\MeshPreviousIndexing;
+use App\Entity\MeshQualifier;
+use App\Entity\MeshTerm;
+use App\Entity\MeshTree;
+use Doctrine\ORM\EntityManager;
+use Doctrine\Persistence\ObjectManager;
+use Liip\TestFixturesBundle\Services\DatabaseToolCollection;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\CoversClass;
+use App\Command\ImportMeshUniverseCommand;
+use App\Repository\MeshDescriptorRepository;
+use App\Service\Index\Mesh;
+use Ilios\MeSH\Parser;
+use Mockery\Adapter\Phpunit\MockeryPHPUnitIntegration;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+
+/**
+ * Acceptance test for the MeSH importer command.
+
+ * @package App\Tests\Command
+ */
+#[Group('cli')]
+#[CoversClass(ImportMeshUniverseCommand::class)]
+class ImportMeshUniverseCommandAcceptanceTest extends KernelTestCase
+{
+    use MockeryPHPUnitIntegration;
+
+    protected CommandTester $commandTester;
+
+    protected MeshDescriptorRepository $meshDescriptorRepository;
+
+    protected EntityManager $entityManager;
+    public function setUp(): void
+    {
+        parent::setUp();
+        $kernel = self::bootKernel();
+        $container = $kernel->getContainer();
+        /** @var EntityManager $em */
+        $em = $container->get('doctrine')->getManager();
+        $this->entityManager = $em;
+
+        // 🧀🧀🧀
+        // Invoke the test fixtures data loader, but without fixtures, to force schema creation in the test db.
+        // @todo Find a solution for this that doesn't involve 🧀. [ST 2025/03/26]
+        $databaseTool = $container->get(DatabaseToolCollection::class)->get();
+        $databaseTool->loadFixtures([]);
```

Comment:
This feels absolutely valid to me as a way to load an empty test db. I'm not sure I understand why it's 🧀.

Reply:
i was looking for a way to explicitly create the schema that doesn't require fixtures loading. couldn't figure it out. 
but will remove the :cheese: from the comment.

Reply:
cheese related comments and TODO removed. :heavy_check_mark: